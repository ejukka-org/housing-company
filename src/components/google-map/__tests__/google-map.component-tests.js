import React from 'react';
import GoogleMap from '../google-map.component';
import renderer from 'react-test-renderer';

test('Renders GoogleMap component', () => {
    const component = renderer.create(
        <GoogleMap
            containerElement={<div style={{ height: `500px`, width: "500px" }} />}
            mapElement={<div style={{ height: `100%` }} />}
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
