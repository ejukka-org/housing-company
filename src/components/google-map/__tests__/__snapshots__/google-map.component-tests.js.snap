// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders GoogleMap component 1`] = `
<div
  style={
    Object {
      "height": "500px",
      "width": "500px",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
  <div />
</div>
`;
